src/resources/material.js

```diff
@@ -36,8 +19,9 @@ function MaterialHandler(app) {
     this._device = app.graphicsDevice;
 
     this._placeholderTextures = null;
-
-    this._parser = new JsonStandardMaterialParser();
+   
+    this._binder = null;
```

Comment:
Assuming a single project is free to mix and match material systems, you will need to instantiate all parsers and binders here (or perhaps lazily) because this single instance of MaterialHandler loads all materials.

Reply:
thanks @slimbuck - latest commit lazily instantiates in arrays of subclass binders/parsers